migration-toolkit-for-containers/migrating-applications-with-mtc.adoc

```diff
@@ -7,6 +7,16 @@ toc::[]
 
 You can migrate your applications by using the {mtc-full} ({mtc-short}) web console or from the command line.
 
+Most cluster-scoped resources are not yet handled by {mtc-short}. If your applications require cluster-scoped resources, you might have to create them manually on the target cluster.
+
+During migration, the {mtc-full} ({mtc-short}) preserves the following namespace annotations:
```

Comment:
If you removed mtc-full in the previous file, friendly reminder to remove it here.